package.json

```diff
@@ -59,13 +62,14 @@
     "eslint-plugin-storybook": "^0.6.12",
     "i18next-scanner": "^4.1.0",
     "sinon-chrome": "^3.0.1",
-    "storybook": "^7.0.12"
+    "storybook": "^7.0.12",
+    "ts-node": "^10.9.1"
   },
   "resolutions": {
     "@polkadot/api": "^10.5.1",
     "@polkadot/keyring": "^12.1.2",
     "@polkadot/networks": "patch:@polkadot/networks@npm:12.1.2#.yarn/patches/@polkadot-networks-npm-12.1.2-012217f8ff.patch",
-    "@polkadot/types": "^10.5.1",
+    "@polkadot/types": "10.5.1",
```

Comment:
The metadata-downloading scripts in this PR were inspired by the ones from azero.dev, but they're using a super-obsolete version of this package. Installing the new one in here was safer, but did cause some inconsistencies in typings so pinning down the version was necessary.